### PR TITLE
Private Events Project - CRR

### PR DESCRIPTION
**Pull request by kevcap**
### Users can:
☑ Login with email
☑ Create events
☑ Attend to events
### Project with
☑ Authentication, Models and Events Management Tested with Rspec + Capybara
☑ Associations between User ↔ Events ↔ Attendees

### CODE REVIEW

**Comment by kevcap** [comment]
## ⚠Disclaimer ⚠
#### Jelil was my partner, but we got unpaired, that's why theres only commits made by me  and why I marked that I worked alone in this project, but added him at the 🏆 Acknowledgement section (README.md).

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @kevincappuccino and @JelilFaisalAbudu 
### A good job you have done so far :clap: However, you have a few issues to solve:-
- [ ] Break down the views into smaller components like in the events/index.html.erb
- [ ] Fix the linting errors in the local environment setup
- [ ] Setup code linters using the GitHub actions [refer to this link](https://github.com/microverseinc/linters-config/tree/master/ror)
### Make the changes and submit for another code review.
- Happy coding!

By Raymond
Slack @iamraymond

File: app/views/events/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                <li class="twitter" style="width:34%;"><a href="#twitter"><span class="fa fa-twitter"></span></a></li>
+                <li class="google-plus" style="width:33%;"><a href="#google-plus"><span class="fa fa-google-plus"></span></a></li>
+              </ul>
+            </div>
+          </li>
+        <% end %>
+      </ul>
+    </div>
+  </div>
+</div>
```
Comment (rOluochKe) [inline app/views/events/index.html.erb]: ### Separate this from a different file, good practice

File: app/views/events/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
```
Comment (rOluochKe) [inline app/views/events/index.html.erb]: ### Add this in a separate file, good practice

**Comment by kevcap** [comment]
Number of linter commits: In my terminal, there was no linter errors, because was a VS Code config, so I wasn't able to find where was the error, because (again) there was no error at the terminal and that error was not a linter rule, at least at the terminal, but it was at the stickler server. Finally I fixed it by changing the end of line from CRLF to LF.

**Review by rOluochKe — APPROVED** [review]
Hello @kevincappuccino and @JelilFaisalAbudu 
### Well done, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit. Good luck with the next one.
- Happy hacking!

**Pull request merged**